packages/bpk-component-checkbox/src/bpk-checkbox.scss

```diff
@@ -41,6 +41,10 @@
     @include bpk-checkbox__input;
   }
 
+  &__icon {
+    @include bpk-checkbox__icon;
```

Comment:
Do we think folk will use this mixin?

Reply:
Dunno, it's something we should discuss for sure. Should we just remove all these mixins at this point?